src/renderer/shared/ui/InfoLink/InfoLink.tsx

```diff
@@ -1,19 +1,30 @@
-import { PropsWithChildren } from 'react';
+import React, { PropsWithChildren } from 'react';
 
 import { cnTw } from '@renderer/shared/lib/utils';
 import { IconNames } from '@renderer/shared/ui/Icon/data';
 import { Icon } from '@renderer/shared/ui';
 
+type AnchorProps = React.DetailedHTMLProps<React.AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>;
```

Comment:
Good practice to provide minimal required  interface
Extending `InfoLink` with all `<a />` props is an overkill
I saw 2 new props `target` which is `_self` [by default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) and `download`
So `{ download?: boolean }` is enough
Don't forget to add condition inside `target="_blank"` if `download is true > target="_self"`